bin/hdfcoinc/pycbc_fit_sngls_split_binned

```diff
@@ -0,0 +1,354 @@
+#!/usr/bin/python
+
+# Copyright 2019 Gareth S. Davies
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+
+from __future__ import division
+
+import sys, h5py
+import argparse, logging
+
+from matplotlib import use
+use('Agg')
+from matplotlib import pyplot as plt
+import copy, numpy as np
+from pycbc import conversions as conv
+from scipy.stats import kstest
```

Comment:
are these two imports used? if not remove

Reply:
Removed - these were a hangover from copying from fit_sngls_binned